Funktionen/Funktionen2.js

```diff
@@ -0,0 +1,35 @@
+const readline = require('readline');
+const rl = readline.createInterface({input: process.stdin, output: process.stdout});
+const prompt = (query) => new Promise((resolve) => rl.question(query, resolve));
+rl.on('close', () => process.exit(0));
+
+async function execute() {
+
+    // const isEven=a;
+    // if (isEven %2===0){
+    //     console.log(true);
+    //     return;
+    // }else {
+    //     console.log(false);
+    //     return;
+    // }
+    // console.log(execute (a ===55))
+    const isEven=(zahl) =>{
+        isEven1 = zahl %2===0?true : false;
```

Comment:
Warum hast du deine Funktion `isEven1` in eine andere Funktion `isEven` hinein gepackt?

Reply:
isEven1 ist  nur name
(isEven1 = zahl %2===0?true : false;
return isEven1 ;)                              ==>  return zahl %2===0?true : false;